nncf/dynamic_graph/graph.py

```diff
@@ -200,9 +200,11 @@ class NNCFNode:
     def __init__(self,
                  node_id: int,
                  op_exec_context: OperationExecutionContext,
-                 module_attributes: BaseModuleAttributes = None):
+                 module_attributes: BaseModuleAttributes = None,
+                 op_type: 'OperatorMetatype' = None):
         self.node_id = node_id
         self.op_exec_context = op_exec_context
+        self.op_type = op_type  # type: OperatorMetatype
```

Comment:
@gadylshintr your PR would probably need to be adjusted so that your `type` attribute is an OperatorMetatype. What do you think?

Reply:
After discussions we decided to keep `OperatorMetatype` local to quantization algo for the time being. The marking of nodes with `OperatorMetatype` values will be done at the quantization algo level, not at the wrap_operator stage.